lib/screens/app/app.dart

```diff
@@ -211,22 +212,33 @@ class _AppState extends State<App> with WidgetsBindingObserver {
     );
   }
 
-  BottomNavigationBarItem buildIcon(String title, String icon, int tabIndex) {
+  BottomNavigationBarItem buildIcon(String title, String icon, int tabIndex, bool profileNotification) {
     final width = MediaQuery.of(context).size.width * 0.21;
     return BottomNavigationBarItem(
-        icon: Container(
-          width: width,
-          decoration: tabIndex == index
-              ? BoxDecoration(
-                  gradient: LinearGradient(colors: AppColors.gradient),
-                  borderRadius: BorderRadius.only(topLeft: Radius.circular(8), topRight: Radius.circular(8)))
-              : BoxDecoration(),
-          padding: EdgeInsets.only(top: 7, left: 3, right: 3),
-          child: SvgPicture.asset(
-            icon,
-            color: tabIndex == index ? Colors.white : AppColors.grey,
+        icon: Stack(children: <Widget>[
+          Container(
+            width: width,
+            decoration: tabIndex == index
+                ? BoxDecoration(
+                    gradient: LinearGradient(colors: AppColors.gradient),
+                    borderRadius: BorderRadius.only(topLeft: Radius.circular(8), topRight: Radius.circular(8)))
+                : BoxDecoration(),
+            padding: EdgeInsets.only(top: 7, left: 3, right: 3),
+            child: SvgPicture.asset(
+              icon,
+              color: tabIndex == index ? Colors.white : AppColors.grey,
+            ),
           ),
-        ),
+          title == "Profile"
+              ? profileNotification
+                  ? Positioned(
+                      child: guardianNotification(profileNotification),
+                      right: 6,
+                      top: 2,
+                    )
+                  : Text("")
```

Comment:
empty Text? 

What are you trying to achieve here?

Reply:
Maybe empty container?